integreat_cms/core/management/commands/duplicate_pages.py

```diff
@@ -86,6 +78,47 @@ def duplicate_pages(region, old_parent=None, new_parent=None):
         duplicate_pages(region, old_parent=old_page, new_parent=new_page)
 
 
-selected_region = Region.objects.first()
-logger.info("Duplicating pages for region %r", selected_region)
-duplicate_pages(selected_region)
+class Command(DebugCommand):
+    """
+    Management command du duplicate all pages of a region
```

Comment:
```suggestion
    Management command du duplicate all pages of a region
```
I think this is a typo. Do you mean "to"?

Reply:
Good catch!